server/graphql/vvm/src/types/vvm_status.rs

```diff
@@ -0,0 +1,51 @@
+use async_graphql::{Object, SimpleObject, Union};
+use repository::vvm_status_row::VVMStatusRow;
+
+#[derive(PartialEq, Debug)]
+pub struct VVMStatusNode {
+    vvm_status: VVMStatusRow,
+}
+
+#[Object]
+impl VVMStatusNode {
+    pub async fn id(&self) -> &str {
+        &self.row().id
+    }
+
```

Comment:
Probably would be good to add the other fields in here, although it can be done later as they're needed.

Reply:
Second this, though thinking that it should be pretty quick to add now?